app/models/label/LabelTable.scala

```diff
@@ -217,6 +231,32 @@ object LabelTable {
   }
 
   /**
+    * Updates the label if it is already in the database, otherwise insert a new one.
```

Comment:
Oops! Thanks for catching this. No it is not used anywhere. This is how I originally implemented the update, but then I decided to change how I do it, but forgot to delete the old function I wrote. Pushing an update now.